modules/gdscript/gdscript.cpp

```diff
@@ -340,12 +341,11 @@ void GDScript::_get_script_property_list(List<PropertyInfo> *r_list, bool p_incl
 			r_list->push_back(E);
 		}
 
-		props.clear();
-
 		if (!p_include_base) {
 			break;
 		}
 
+		props.clear();
```

Comment:
Moving this is a superficial change, right? Doesn't seem like it would affect anything.

Reply:
Correct, it's just more in line with other methods :)